library/src/webapp/js/spinner.js

```diff
@@ -209,7 +209,9 @@ SPNR.disableSelects = function( clickedElement, escapeList )
 
             // Create the cloned element and disable it
             var newSelect = document.createElement( "select" );
-            newSelect.setAttribute( "id", select.getAttribute( "id" ) + "Disabled" );
+            var oldId = select.getAttribute( "id" );
```

Comment:
Just curious if the old ids were a reference to anything?

Reply:
Yeah, I thought something similar when I found this spinner code. I guess it's always executed on a page refresh so it should be fine. It's been like that for a while and there haven't been related issues, so I just focused on this specific bug and didn't modify anything else.